### ISSUE TITLE
Venue Activity Tabs need labels to match "Nearby" and "Friends" tabs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#57]
```
What steps will reproduce the problem?
1. Click on ah venue
2. Notice that the tabs don't have labels in the same fashion as the "Nearby" 
and "Friends" tab

What is the expected output? What do you see instead?
All tabs should have consistent labels.

Please use labels and text to provide additional information.
Can this be thrown into the strings.xml file?

```

Original issue reported on code.google.com by `ChrisBru...@gmail.com` on 26 Aug 2009 at 3:47

**Comment by GoogleCodeExporter** [comment]
```
It can't /just/ be thrown into sttrings.xml but can be done easily if that is a 
good odea. If you want labels 
for tabs, find the right TabActivity and update the init tab methods.
```

Original comment by `jlapenna@google.com` on 28 Aug 2009 at 8:19

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision f66689393e.
```

Original comment by `jlapenna` on 29 Aug 2009 at 7:03
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision f66689393e.
```

Original comment by `jlapenna` on 5 Jan 2010 at 5:47
